ui/app/styles/core/element-styling.scss

```diff
@@ -171,9 +171,26 @@ label {
   cursor: pointer;
 }
 
-// HDS modifications
+// HDS modifications and overrides
```

Comment:
Thinking this could move elsewhere eventually if we collect a lot of them. For now I wanted them in an easy to find location because these are classes already attached to hds components, as in, they're not searchable in the codebase by classname